preferences/src/main/java/com/itsaky/androidide/preferences/IDEPreferences.kt

```
@@ -19,26 +19,34 @@ package com.itsaky.androidide.preferences
+	// Never actually read at runtime today - PreferencesActivity passes the literal
+	// TooltipTag.PREFS_TOP constant for the root fragment instead of this property - but this is
+	// the root container this constant represents, so give it the matching value rather than an
+	// unused empty string.
+	override val tooltipTag: String = TooltipTag.PREFS_TOP
+
+	override fun describeContents(): Int = 0
+
+	override fun writeToParcel(
+		dest: Parcel,
+		flags: Int,
+	) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)